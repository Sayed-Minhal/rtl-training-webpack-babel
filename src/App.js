import React from 'react';
import SimpleComponent from './components/SimpleComponent';

function App() {
    return <SimpleComponent />;
}

export default App;