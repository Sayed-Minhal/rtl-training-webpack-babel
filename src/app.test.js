import React from "react";
import { render, screen } from "@testing-library/react";

import App from './App';

describe('Render component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = render(<App />);
    });

    it('should take snapshot', () => {    
        expect(wrapper.container).toMatchSnapshot();
    });
    
    it('should find the title', () => {
        const titleElement = screen.getByText(/Ya Ali Madad/i);
        expect(titleElement).toBeInTheDocument();
    });
});
