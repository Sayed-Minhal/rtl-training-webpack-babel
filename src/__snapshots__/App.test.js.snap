// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component should take snapshot 1`] = `
<div>
  Ya Ali Madad!
</div>
`;
